ranking-service/src/main/resources/application.yml

```
@@ -0,0 +1,18 @@
+      - classpath:./properties/datasource.yml
+      - classpath:./properties/jpa.yml
+  profiles:
+    group:
+      local: local, key
+    active: ${ACTIVE_VAL}
+eureka:
+  client:
+    service-url:
+      defaultZone: ${EUREKA_SERVER}  # Eureka 서버 URL
+server:
+  port: ${RANKING_SERVER}
```

YAMLlint (1.35.1)

[error] no new line character at the end of file

(new-line-at-end-of-file)